starter-core/src/test/groovy/io/micronaut/starter/feature/graalvm/GraalNativeImageSpec.groovy

```diff
@@ -283,22 +306,30 @@ class GraalNativeImageSpec extends BeanContextSpec implements CommandOutputFixtu
         dockerfile.contains('RUN zip -j function.zip bootstrap foo')
         dockerfile.contains('ENTRYPOINT ["/home/application/foo"]')
 
-
-        and: 'defaults to graalvm ce jdk8 image'
-        !dockerfile.contains('FROM oracle/graalvm-ce:20.1.0-java8 as graalvm')
-        !dockerfile.contains('#FROM oracle/graalvm-ce:20.1.0-java11 as graalvm')
+        and: 'different graalvm image depending on JDK version'
+        if (jdkVersion == JdkVersion.JDK_8) {
+            dockerfile.contains('FROM oracle/graalvm-ce:20.1.0-java8 as graalvm')
```

Comment:
missing `assert`.

Reply:
Good catch! I don't know why that's necessary because the statement is in a `then` block.